NickvisionTagger.GNOME/Views/MainWindow.cs

```diff
@@ -1233,4 +1236,23 @@ private void CopyFingerprintToClipboard(Gtk.Button sender, EventArgs e)
         _fingerprintLabel.GetClipboard().SetText(_fingerprintLabel.GetText());
         _toastOverlay.AddToast(Adw.Toast.New(_("Fingerprint was copied to clipboard.")));
     }
+
+    /// <summary>
+    /// Occurs when there are corrupted music files found in a music folder
+    /// </summary>
+    private bool CorruptedFilesFound()
+    {
+        var dialog = new MessageDialog(this, _controller.AppInfo.ID, _("Corrupted Files Found"), _("This music folder contains music files whose tags are corrupted. What would you like to do?"), _("Ignore Files"), _("Clear Tags"));
```

Comment:
How about "...What would you like to do **with corrupted files**?" so it's understandable that only corrupted tags will be cleared/ignored.

Reply:
Agreed :)

Could you make the change and update the po files?

Reply:
okay, sure :)